index.js

```diff
@@ -425,4 +429,18 @@
     return JSON.parse(s);
   });
 
+  //  regexp  /////////////////////////////////////////////////////////////////
```

Comment:
Yeah, that one was a bit puzzling and then I realised I'd added another space as well as the 'p' character!